CARTOGRAPHY.md

```diff
@@ -37,3 +37,53 @@ account.
 * **Adaptability and ease of use** - The style should be easy to customize, like for creating localized or special purpose maps. It is also important to keep demands on rendering infrastructure for serving the style low so it is not too difficult and costly to set up a tile server for this style or a specialized variant of it.
+
+## Design guidelines
+
+The following guidelines have been found to be useful design principles that can help creating a better map in line with the above goals.  Keep in mind that none of these principles represents a goal on its own and most of them are highly condensed summaries of more complex considerations.
+
+* The difference in rendering between different types of features should foremost be based on their difference in meaning and purpose for the target map users.  Differences in physical appearance are only relevant as far as they also represent differences in meaning and purpose.
+
+* Design of symbols, area and line patterns should be in a way that avoids their geometries to be mistaken as actual data.
+
+* The connection between data and appearance of features in the map needs to be comprehensible for the viewer.
+
+* When changes have a strong impact on the overall appearance of the map input should be sought from the broader OpenStreetMap community and particular consideration given to their comments and concerns.
+
+### Colors
+
+* Differentiating features through different colors only makes sense if this difference can actually be read by the map users.
+
+* For area colors we prefer light and low saturation tones.  Lines, individual symbols and labels also can be stronger in color.
+
+* Stronger colors indicate fairly distinct and meaningful features as opposed to more general and less distinct features which use weaker colors.
+
+* Small areas work well with somewhat stronger colors while large areas tend to call for weaker tones.
+
+* Symbols of area patterns need to be relatively weak in color with little contrast towards the base color so they cannot be mistaken for an individual symbol.
+
+* Area outlines should - when used - be in colors that are not too strong and harmonic with the fill color.  Care should be taken that area outlines are not mistaken for line features.
+
+* Use of blue for area and line colors should be reserved for water related features.  Symbols and labels in blue color can also be used for other purposes but sparsely and in a tone clearly distinct from all water related features
+
+* Use of green should be reserved for vegetation related features, in particular in case of area and line colors.
+
+* Use of red, orange, yellow and brown line colors should be reserved to roads and paths.
+
+* Transparency tends to lead to undesirable and ambiguous color mixing and should therefore be used very carefully.
```

Comment:
The most prominent use of transparency is currently the boundaries.  This is especially problematic since they are in strong color leading to strange color mixing.  Other uses are nature reserves and military areas.